src/lib.rs

```diff
@@ -20,6 +20,7 @@
 //! So things like [String] will work out of the box.
 //!
 //! TODO! This is all wrong. Need to talk about VS, not Version.
+//! 
 //! Our Version trait requires
```

Comment:
This paragraph is out of date and needs to be rewritten. I'm open to suggestions for how to explain `VS`.